src/components/dropdowns/Dropdown.tsx

```diff
@@ -49,12 +49,12 @@ export function Dropdown<T>({
               leaveFrom="opacity-100"
               leaveTo="opacity-0"
             >
-              <ListboxOptions className="absolute z-10 mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
+              <ListboxOptions className="absolute z-10 mt-1 max-h-60 overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
                 {options.map((option, index) => (
                   <ListboxOption
                     key={index}
                     className={
-                      "relative cursor-pointer select-none py-2 pl-3 pr-9 text-gray-900 hover:bg-wsg-orange-700"
+                      "relative cursor-pointer select-none px-4 py-2 text-gray-900 hover:bg-wsg-orange-700"
```

Comment:
I can't explain why, but somehow it looked better before.

Reply:
I found it a bit weird here:

![image](https://github.com/user-attachments/assets/9f597e8d-752c-4e63-a61f-59f1cf52da8f)

But I get what you mean in the other places and here in German, so I reverted it